src/code/z_collision_check.c

```diff
@@ -2153,32 +2153,32 @@ void CollisionCheck_AC_CylVsTris(GlobalContext* globalCtx, CollisionCheckContext
  */
 void CollisionCheck_AC_TrisVsCyl(GlobalContext* globalCtx, CollisionCheckContext* colChkCtx, Collider* colAT,
                                  Collider* colAC) {
-    static Vec3f D_8015E310;
+    static Vec3f intersection;
     ColliderTris* at = (ColliderTris*)colAT;
     ColliderTrisElement* atItem;
     ColliderCylinder* ac = (ColliderCylinder*)colAC;
     Vec3f atPos;
     Vec3f acPos;
 
     if (ac->dim.radius > 0 && ac->dim.height > 0 && at->count > 0 && at->elements != NULL) {
-        if (CollisionCheck_SkipBump(&ac->info) == 1) {
+        if (CollisionCheck_SkipBump(&ac->info) == true) {
             return;
         }
         for (atItem = at->elements; atItem < at->elements + at->count; atItem++) {
-            if (CollisionCheck_SkipTouch(&atItem->info) == 1) {
+            if (CollisionCheck_SkipTouch(&atItem->info) == true) {
                 continue;
             }
-            if (CollisionCheck_NoSharedFlags(&atItem->info, &ac->info) == 1) {
+            if (CollisionCheck_NoSharedFlags(&atItem->info, &ac->info) == true) {
                 continue;
             }
 
-            if (Math3D_CylTriVsIntersect(&ac->dim, &atItem->dim, &D_8015E310) == 1) {
+            if (Math3D_CylTriVsIntersect(&ac->dim, &atItem->dim, &intersection) == true) {
```

Comment:
`intersection` is called `hitPos` in `CollisionCheck_AC_CylVsTris`, should probably go with one or the other

Reply:
intersection is probably the more math-correct term (and it goes with the function name), but will say in a vaccum I think hitPos will resonate more with the non-math folks. mainly cause it clarifies that its a position instead of implying it. intersectPos is a possibility i guess, but may be overly verbose.

just my personal opinion, if yall think intersection is better then by all means

Reply:
No need to use the more general `intersection` when the variable has a clear semantic meaning in the context of the function IMO.